Bottom: d5c66e49f2f95d0fd1eccbaa069a95bf1743a936
Top:    ad5ed3896ce9dafe9157f24db449f2f005d272ab
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-19 00:25:01 +0000

Refresh of play-with-some-lighting-effect

---

diff --git a/src/bumps.png b/src/bumps.png
deleted file mode 120000
index c0b7227..0000000
--- a/src/bumps.png
+++ /dev/null
@@ -1 +0,0 @@
-smooth.png
\ No newline at end of file
